### PR TITLE
Bump version to v2021-08-11

### PR DESCRIPTION
**Pull request by Anviking**
- [x] Bump wallet version to v2021-08-10 in preparation for release

### Issue Number

Release.

### Comments

Dependent on #2811

### CODE REVIEW

File: scripts/make_release.sh [hunk 1]
```diff
@@ -19,10 +19,9 @@ set -euo pipefail
 ################################################################################
 # Release-specific parameters (Change when you bump the version)
 # Release tags must follow format vYYYY-MM-DD.
-GIT_TAG="v2021-07-30"
-OLD_GIT_TAG="v2021-06-11"
-
-CARDANO_NODE_TAG="1.28.0"
+GIT_TAG="v2021-08-10"
```
Comment (rvl) [inline scripts/make_release.sh]: Could we make it v2021-08-11, because we will probably only fully finish it tomorrow.

**Review by rvl — APPROVED** [review]

File: scripts/make_release.sh (hunk 1 above)
Comment (Anviking) [reply scripts/make_release.sh]: 👍

**Pull request merged**